lib/components/inputs/UploadField/UploadField.tsx

```diff
@@ -82,6 +87,17 @@ function UploadField(props: UploadFieldProps) {
   return (
     <Tooltip data={tooltipText}>
       <div className={uploadWrapperClasses}>
+        {description && (
+          <span className='field__description-wrap'>
+            <span className='field__description field-1-description-content'>
+              {description}
+              {isRequired && <span className='required-star'>*</span>}
```

Comment:
the star looks really small, let's not change its size:

before:
![Screenshot 2024-11-27 at 16 26 11](https://github.com/user-attachments/assets/2a9ed763-a870-48ce-ae19-325e188931f5)

after:
![Screenshot 2024-11-27 at 16 26 19](https://github.com/user-attachments/assets/bb6794e0-dd8f-426f-a992-862dabb4cf55)